orderbook/src/metrics.rs

```diff
@@ -14,6 +14,8 @@ pub struct Metrics {
     db_table_row_count: IntGaugeVec,
     /// Outgoing RPC request metrics
     rpc_requests: HistogramVec,
+    pool_cache_hits: IntCounter,
```

Comment:
Nanonit: could also use one metric with two labels to keep this struct a bit smaller (feel free to ignore)

Reply:
Thought about that but I find it more natural and in line with how Prometheus docs describe new name vs new label https://prometheus.io/docs/practices/naming/ .